### DISCUSSION TITLE
Quickstart

### DISCUSSION THREAD

**Discussion by emmanuelha** [linkedin/Hoptimator#58] [Q&A]
Hello, 
I would like some more information about how to quickstart using Hoptimator.
First of all when you run:
> make quickstart

You do not start hoptimator container so when you run:
> ./bin/hoptimator

It can not enter into hoptimator container because it does not exist.  
So I guess we should run deploy-samples and use instead:
> make bounce

?

Then when I do that and that I launch sqlline again I got:
 [main] WARN org.apache.kafka.clients.ClientUtils - Couldn't resolve server one-kafka-bootstrap.kafka.svc:9092 from bootstrap.servers as DNS resolution failed for one-kafka-bootstrap.kafka.svc
 Where does 'one-kafka-bootstrap.kafka.svc' for the bootstrap.servers comes from ?

**Comment by emmanuelhar** [comment]
Ok I think I understood:
In the makefile the task deploy-dev-environment is wrong because the flink operator repo address https://downloads.apache.org/flink/flink-kubernetes-operator-1.4.0/ does not exists (anymore ?). It should be replaced with 1.6.1 or 1.7.0.
So line 31 of the makefile was not executed and no kafka pod was created.
By the way once in sqlline the consumed cpu went to the sky so I could not execute the sql query :=( ...

**Reply by ryannedolan** [reply]
Good catch, looks like Strimzi no longer supports the version of Kafka I have cluster "one" set up to use. We need to either 1) delete the `version` from the Kafka cluster yaml, and let Strimzi pick a default, or 2) pin the version of Strimzi to an older version (instead of `latest`). I tried (1) in #57 and it seems to work, but I don't know if it's a good long-term strategy. Probably better to pin a specific version.

Care to submit a PR?

**Comment by emmanuelhar** [comment]
Yes I can create a PR but to be sure that I understood correctly:
I think the problem is with the link to 1.4.0 of the operator https://downloads.apache.org/flink/flink-kubernetes-operator-1.4.0/ which is not correct anymore.   
From the archive directories I see: https://archive.apache.org/dist/flink/flink-kubernetes-operator-1.4.0/
So I suggest to change only this url.
If I do that and then launch:
`make quickstart`
Which when running (I added a debut flag):

`helm upgrade --install --debug --atomic --set webhook.create=false flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator`

=> gave me after 5 minutes looping:
> ready.go:281: [debug] Deployment is not ready: default/flink-kubernetes-operator. 0 out of 1 expected pods are ready 
 
Then the script failed:
> release flink-kubernetes-operator failed, and has been uninstalled due to atomic being set  

Do you have any idea ?

**Reply by ryannedolan** [reply]
Oh maybe there are two issues? I'm aware only of the version conflict wrt Strimzi (Kafka operator). The Kafka cluster is set up to use a version which the latest Strimzi no longer supports. It's possible the same or similar issue exists with the Flink operator.
